Docker.md

```diff
@@ -86,47 +85,48 @@ The image contains the following directories:
-| /usr/local/tomcat/conf/server.xml                  | mount/copy of your server.xml                        | Contains the default server.xml of Tomcat, replace to secure your application                                                                                            |
-| /usr/local/tomcat/conf/catalina.properties         | Server properties, contains default framework values | Do not replace this file, use [Environment variables](#Environment-variables) or append to the file, see [Dockerfile](docker/appserver/Tomcat/Dockerfile) for an example |
+| file                                       | description                                          | notes                                                                                                                                                                    |
+|--------------------------------------------|------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| /opt/frank/resources/resources.yml         | mount/copy of your resources.yml                     | Use hostname `host.docker.internal` to get to the host machine for local testing. Changing this file will require a new instance to be started, it cannot be reloaded    |
+| /usr/local/tomcat/conf/server.xml          | mount/copy of your server.xml                        | Contains the default server.xml of Tomcat, replace to secure your application                                                                                            |
+| /usr/local/tomcat/conf/catalina.properties | Server properties, contains default framework values | Do not replace this file, use [Environment variables](#Environment-variables) or append to the file, see [Dockerfile](docker/appserver/Tomcat/Dockerfile) for an example |
 
 ## Logging
 
 Generated log files are stored in `/usr/local/tomcat/logs`.
 
+In some cases you might want to change the log appenders, for example to log to stdout. Refer to the [Frank!Framework Manual: Custom Logging](https://frank-manual.readthedocs.io/en/latest/deploying/customLogging.html#custom-logging) for more information.
+
 ## Environment variables
 
 Environment variables can be used to set properties. Environment variables have the highest precedence and override
 application properties set in .property files supplied by Tomcat, resources and configurations.
 
-Environment variables can also be used to replace properties set inside Tomcat configuration files such as `server.xml` and `context.xml`.
+Environment variables can also be used to replace properties set inside Tomcat configuration files such as `resources.yml` and `server.xml`.
```

Comment:
```suggestion
Though you probably do not need to touch these file, because of the CredentialProvider, environment variables can also be used to replace properties set inside Tomcat specific configuration files such as `server.xml` and `context.xml`.
```